moveit_ros/planning/plan_execution/include/moveit/plan_execution/plan_representation.h

```diff
@@ -82,6 +82,12 @@ struct ExecutableMotionPlan
 
   /// An error code reflecting what went wrong (if anything)
   moveit_msgs::MoveItErrorCodes error_code_;
+
+  planning_scene::PlanningScenePtr copyPlanningScene()
+  {
+    planning_scene_monitor::LockedPlanningSceneRO ls(planning_scene_monitor_);
+    return planning_scene_->diff();
```

Comment:
This is no copy yet!

```suggestion
    planning_scene::PlanningScenePtr copy = planning_scene_->diff();
    copy.decoupleParent();
    return copy;
```

Reply:
I was wondering if this was needed. I've replaced the `diff` calls with `clone`, as that seems to do the right thing.